File: Bridge/System/Runtime/Serialization/SerializationException.cs
using Bridge;

namespace System.Runtime.Serialization
{
    public class SerializationException : SystemException
    {
        // Constructors
        public SerializationException()
            : base("An error occurred during (de)serialization")
        {
        }

        public SerializationException(string message)
            : base(message)
        {
        }

        public SerializationException(string message, Exception innerException)
            : base(message, innerException)
        {
        }
    }
}

File: Bridge/Resources/.generated/system/runtime/serialization/serializationException.js
    Bridge.define("System.Runtime.Serialization.SerializationException", {
        inherits: [System.SystemException],
        ctors: {
            ctor: function () {
                this.$initialize();
                System.SystemException.ctor.call(this, "An error occurred during (de)serialization");
            },
            $ctor1: function (message) {
                this.$initialize();
                System.SystemException.ctor.call(this, message);
            },
            $ctor2: function (message, innerException) {
                this.$initialize();
                System.SystemException.ctor.call(this, message, innerException);
            }
        }
    });
